src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,66 @@
+.projects-listing {
+display: grid;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)